### DISCUSSION TITLE
왼쪽 사이드바가 계정 아이콘을 눌러야만 뜹니다.

### DISCUSSION THREAD

**Discussion by IN-JIWON** [pronist/hELLO#216] [Q&A] [answered]
전에는 안그랬는데 갑자기 최근에 왼쪽 사이드바가 바로 보이지 않고 계정 아이콘 사진을 눌러야 뜹니다.
아무것도 만진 것이 없는데 갑자기 바뀌었습니다.
어떻게 해결하면 좋을 까요?
![image](https://github.com/pronist/tistory-skin-hELLO/assets/73393886/547891e0-e952-490b-b5ae-e2381e3bf418)
![image](https://github.com/pronist/tistory-skin-hELLO/assets/73393886/323855a1-a58a-4acf-abb1-a82f218f2b77)

왼쪽 사이드바를 고정하고 싶다는 기존 질문글을 확인하였는데 거기도 별다른 해결 방법 없이 질문자가 이해했다는 댓글만 있어 따로 질문글을 올립니다.

**Comment by pronist** [comment, marked as answer]
@IN-JIWON 안녕하세요.

사이드바의 경우 가로 `1400px` 를 기준으로 On/Off 가 결정됩니다. 또한 **관리자 - 스킨편집**에서 사이드바와 관련된 옵션이 On 으로 되어 있는지 점검 부탁드립니다. 그림으로 추측하건데 리스트와 양옆 여백의 상태로 보아하니 가로 사이즈의 문제일 것으로 판단됩니다. 가로 사이즈가 문제되는 경우 현재 사용하고 계신 환경에 따라 반응하여 스킨의 레이아웃이 바뀐 것이므로 이는 버그가 아닌 스킨의 사양인지라 해결책을 제시해드릴 수 없습니다.

![image](https://github.com/pronist/tistory-skin-hELLO/assets/25624965/c3e9809a-e3f1-4ab3-b538-8ce9de462e5a)
